### ISSUE TITLE
write_attributes / attributes= method

### ISSUE THREAD

**Issue by etipton** [Dynamoid/dynamoid#426]
Thoughts on adding something like this (possibly naming it `attributes=`)?

```
    # Analagous to ActiveRecord's #attributes= -- allows for writing attributes via a hash
    def write_attributes(attrs)
      attrs.each do |attr, val|
        write_attribute(attr, val)
      end
    end
```

**Comment by andrykonchin** [comment]
Yes, it's a good idea to follow the Rails naming conventions and PR is welcomed.

But unfortunately there is already `attributes` accessor (defined [here](https://github.com/Dynamoid/dynamoid/blob/master/lib/dynamoid/fields.rb#L145)). It's too low-level and doesn't cast attribute values. So **it makes sense** to change it with proposed implementation. But it would be a breaking change so I believe we should wait until the next major release.

There is similar method `#load` ([here](https://github.com/Dynamoid/dynamoid/blob/master/lib/dynamoid/loadable.rb#L7)) which is used completely internally but accidentally public. It could be used for now and later in the next major release it will become private.

**Comment by etipton** [comment]
Thanks for the quick response!
